test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -1147,4 +1150,60 @@ UCS_TEST_P(test_ucp_tag_stats, rndv_unexpected, "RNDV_THRESH=1000") {
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_tag_stats)
 
+class multi_rail_max : public test_ucp_tag_xfer {
+public:
+    void init()
+    {
+        stats_activate();
+        test_ucp_tag_xfer::init();
+    }
+
+    void cleanup()
+    {
+        test_ucp_tag_xfer::cleanup();
+        stats_restore();
+    }
+};
+
+UCS_TEST_P(multi_rail_max, max_lanes, "IB_NUM_PATHS?=8", "TM_SW_RNDV=y")
+{
+    /* MAX_RNDV_LANES must be initialized after 'init' */
+    modify_config("MAX_RNDV_LANES", "8");
```

Comment:
can't it be in test header? line 1168

Reply:
no, because will be overridden by test::init